Source/WTF/wtf/PlatformHave.h

```diff
@@ -1618,3 +1618,7 @@
     || (PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 170000)
 #define HAVE_AUDIOFORMATPROPERTY_VARIABLEPACKET_SUPPORTED 1
 #endif
+
+#if PLATFORM(IOS) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 110000
```

Comment:
I don't think we need the `MIN_REQUIRED` since we don't support that far back any more.